heron/scheduler-core/src/java/com/twitter/heron/scheduler/utils/SchedulerUtils.java

```diff
@@ -369,26 +369,13 @@ public static String decodeJavaOpts(String encodedJavaOpts) {
   }
 
   /**
-   * Setup the working directory:
-   * <br> 1. Download heron core and the topology packages into topology working directory,
-   * <br> 2. Extract heron core and the topology packages
-   * <br> 3. Remove the downloaded heron core and the topology packages
+   * Setup the working directory: Create the directory if it does not exist
+   * otherwise clean the directory.
    *
    * @param workingDirectory the working directory to setup
-   * @param coreReleasePackageURL the URL of core release package
-   * @param coreReleaseDestination the destination of the core release package fetched
-   * @param topologyPackageURL the URL of heron topology release package
-   * @param topologyPackageDestination the destination of heron topology release package fetched
-   * @param isVerbose display verbose output or not
    * @return true if successful
    */
-  public static boolean setupWorkingDirectory(
```

Comment:
this method no longer sets up directory, should rename it peoperly

Reply:
I will rename it to something better.